recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ requirements:
     - matplotlib >=1.3.1
     - networkx >=1.8
     - pillow >=2.1.0
-    - dask >=0.5
+    - dask-core >=0.15
+    - toolz >=0.7.4
```

Comment:
Does [`toolz >=0.7.3`]( https://github.com/dask/dask/blob/0.15.0/setup.py#L10 ) work or is there another reason for `0.7.4`?

Reply:
0.7.4 is the earliest version available in conda-forge, so I didn't see a reason to put the earlier version in!

Reply:
Thanks for the clarification.